src/app/components/media-viewer-wrapper/media-viewer-wrapper.component.spec.ts

```diff
@@ -9,6 +9,7 @@ import { StoreModule } from '@ngrx/store';
 import { EffectsModule } from '@ngrx/effects';
 import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
 import { RouterTestingModule } from '@angular/router/testing';
+import { SessionStorageService } from 'src/app/services';
```

Comment:
Shouldn't import from `src/`.

Reply:
Changed to relative path. Still waiting for Phil's path change to go in...